### PR TITLE
[IMP] accounting: checks

### PR DESCRIPTION
**Pull request by toaa-odoo**
Cleared up some confusion about path menu and feature

Forward to master

taskid-3644544

### CODE REVIEW

File: content/applications/finance/accounting/payments/pay_checks.rst [hunk 1]
```diff
@@ -13,9 +13,10 @@ Activate checks payment methods
 -------------------------------
 
 To activate the checks payment method, go to :menuselection:`Accounting --> Configuration -->
-Checks`, there you can activate the payment method as well as set up the :guilabel:`Check Layout`.
-Once activated the :guilabel:`Checks` setting, the feature is automatically setup for your
-:guilabel:`Outgoing Payments`.
+Settings`, and scroll down to the :guilabel:`Vendor Payments` section. There, you can activate the
+payment method as well as set up the :guilabel:`Check Layout`. Once the :guilabel:`Checks` setting
+is activated, the **Checks** payment method is automatically set up in the
+:guilabel:`Outgoing Payments` in **bank** journals.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/payments/pay_checks.rst:19]:
```suggestion
:guilabel:`Outgoing Payments` tabs of **bank** journals.
```
Suggestion

**Review by xpl-odoo — APPROVED** [review]
Just a suggestion!
@robodoo delegate +

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hi @toaa-odoo just a tiny little change :-)

**Review by auva-odoo — APPROVED** [review]

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**